src/main/scala/TooHot.scala

```diff
@@ -0,0 +1,9 @@
+object TooHot extends App{
+
+  def climate (temperature:Int, isSummer:Boolean):Unit = {
+    if (isSummer==true && temperature > 60 && temperature<=100) print("true")
+    else if (isSummer==false && temperature >60 && temperature<=90 ) print("true")
```

Comment:
!isSummer could replace isSummer==false

Reply:
Ok, thanks. I didn't know that you could use this syntax